pypeit/core/skysub.py

```diff
@@ -1108,16 +1111,18 @@ def ech_local_skysub_extract(sciimg, sciivar, fullmask, tilts, waveimg, global_s
     return skymodel, objmodel, ivarmodel, outmask, sobjs
 
 
-def read_userregions(text, resolution=1000):
+def read_userregions(skyreg, nslits, resolution=1000):
     """ Parse the sky regions defined by the user. The text should
         be a comma separated list of percentages to apply to all slits
         Example: The following string   :10,35:65,80:
         would select the first 10%, the inner 30%, and the final 20% of all slits
 
     Parameters
     ----------
-    text : str
+    skyreg : str
         The sky region definition.
+    nslits : int
+        Number of slits on the detector
     resolution: int, optional
```

Comment:
I cannot follow your documentation. I don't understand what resolution is or why it is 1000. I think specifying percentages is bad news. Why not use absolute pixel coordinates at some reference (y, i.e. spectral) location, since that is what the user sees in the GUI. Then you can compute the percentages internally, but the user is pointing to a specific pixel. Perhaps I'm not following your procedure.